Content.Server/Xenoarchaeology/Artifact/XAT/Components/XATMagnetComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Server.Xenoarchaeology.Artifact.XAT.Components;
+
+[RegisterComponent, Access(typeof(XATMagnetSystem))]
+public sealed partial class XATMagnetComponent : Component
+{
+    /// <summary>
+    /// how close to the magnet do you have to be?
+    /// </summary>
+    [DataField]
+    public float MagnetRange = 40f;
```

Comment:
i'm kinda not sure why is it here - shouldn't magnet effectiveness be part of magnet related code? And magnet and magboots creating magnet field... whatever, let it be.